src/wallet/rpcdump.cpp

```diff
@@ -100,6 +100,7 @@ UniValue importprivkey(const JSONRPCRequest& request)
         throw std::runtime_error(
             "importprivkey \"privkey\" ( \"label\" ) ( rescan )\n"
             "\nAdds a private key (as returned by dumpprivkey) to your wallet. Requires a new wallet backup.\n"
+            "\nHint: use importmulti to import more than one private key!\n"
```

Comment:
Style nits. Feel free to ignore:

- doesn't require a blank line between `Adds a private key...` and `Hint: use...`
- remove the `!`. `.` will do!

Reply:
Done!